### PR TITLE
Group create list

### PR DESCRIPTION
**Pull request by DhariniJeeva**

### CODE REVIEW

File: addons/api/addon/abilities/collection.js [hunk 1]
```diff
@@ -29,7 +29,8 @@ export default class CollectionAbility extends Ability {
    * @return {boolean}
    */
   hasAuthorizedCollectionAction(action) {
-    const { authorized_collection_actions } = this.model;
+    const authorized_collection_actions =
+      (this.model || {}).authorized_collection_actions || {};
```
Comment (gsusmi) [inline addons/api/addon/abilities/collection.js:33]: Thoughts on using optional chaining and using possible `undefined` return value to check if `action` include should be verified?
```
const authorized_collection_actions = this?.model?.authorized_collection_actions
```

File: ui/admin/tests/acceptance/groups/list-test.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    instances.group = this.server.create('group', {
+      scope: instances.orgScope,
+    });
+    orgURL = `/scopes/${instances.orgScope.id}`;
+
+    urls.groups = `/scopes/${instances.orgScope.id}/groups`;
+    authenticateSession({});
+  });
+
+  test('can navigate to groups with proper authorization', async function (assert) {
```
Comment (calcaide) [inline ui/admin/tests/acceptance/groups/list-test.js:51]: For consistency with other test cases, missing "User" on the test description, just very minor stuff.

**Review by calcaide — APPROVED** [review]
LGTM! Awesome job!

**Review by gsusmi — APPROVED** [review]
LGTM. Ty!

**Pull request merged**